_extras/18-nextsteps.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Bootstrapping your use of HPC"
+teaching: 10
+exercises: 60
+questions:
+- "How can I get started on using HPC?"
+- "Where can I get help to start using HPC?"
+objectives:
+- "Get help to get your work up and running on an HPC system"
+- "Understand where you can get help from in the future"
+keypoints:
+- "Understand the next steps for you in using HPC."
+- "Understand how you can access help and support to use HPC."
+---
+
+Now you know enough about HPC to explore how to use it for your work or to understand
+what its potential benefits are you. You may also have ideas around where the 
+barriers and difficulties may lie and have further questions on how you can 
+start using and/or trying HPC in your area.
+
+This session is designed to give you the opportunity to explore these questions and
+issues. The instructors and helpers on the course will be on hand to answer your
+questions and discuss next steps with you.
+
+> ## Potential discussions
+>
+> Things you could discuss with the instructors and helpers could include:
+>
+> - Your computational workflow and where advanced computing could help
+> - How to get access to facilities for your work
+> - How to get help and support to get your work running using advanced computing.
+>   For example, software development, further training, access to local expertise
+{: .callout}
+
+## Options for this session
+
+There are a number of different options for practical work during this session. The
+challenges below include: exploring your own work; an extended example using a parallel
+HPC application; an extended example using high throughput computing on multiple
+serial analyses. If you have something else you want to use the session for (e.g. to
+discuss things with the instructors/helpers as described above) then please feel free
+to do this. The idea of the session is to help you bootstrap your use of advanced computing
+and this will differ from individual to individual!
+
+> ## Exploring your work using HPC
+>
+> If you have a practical example of something from your area of work that you would like
+> help with getting up and running on an HPC system or exploring the performance of
+> on an HPC system, this is great! Please feel free to discuss this with us and ask
+> questions (both technical and non-technical).
+{: .challenge}
+
+> ## Exploring the performance of GROMACS
```

Comment:
This is a very life-science based exercise and it is very involved. I think this is worth a capstone project. I would call it as such. Moreover, for people that have no professional relationship with Gromacs, the outcome of the exercise might be irrelevant and thus demotivating.

Reply:
This is why it is in `_extras` and not part of the main lesson. This was a useful resource that I created for a particular delivery of HPC Carpentry and thought it might be useful more widely but did not fit within the main lesson.

If you think the inclusion in `_extras` is still too strong I am happy to take it out.

I could bring it back as its own single lesson ('hpc-bootstrap'?) that could be appended to other lessons as needed.

I would be interested in your opinion here.